Cloud Application Manager/Deploying Anywhere/orchestrating-vcloud-air-vcloud-director-deployments.md

```diff
@@ -57,37 +69,57 @@ Create a [deployment policy](../Automating Deployments/deploymentpolicy-box.md)
-| Catalog |	Select the catalog that contains the vApp template for deployment. The catalog stores vApp templates and media files. Select from shared, private, and public catalogs that your organization can access. |
-| vApp Template | Select a vApp template from the catalog. VMs imported as OVFs then uploaded as templates are also listed. |
-| Customization Password | Optionally, provide a custom VM administrator password. If you leave this empty, the default administrator password from the template applies.|
-| Instances | Specify the number of instances to spawn from the same template. |
+When creating an instance, an automatic name is generated and used for VApps, VMs and Hostname, all with the same generated name. This way you can identify unregistered instances.
 
-**Placement**
+This name generated has maximum length of 15 characters (max allowed in windows hostnames), and it's composed by this values:
 
-| Option | Description |
-|--------|-------------|
-| Virtual Data Center |	Select the virtual data center from vCloud Air or vCloud Director. |
-| Network |	Select the network for the virtual data center. |
-| IP Allocation | Assign IP addresses to the VM from the network attached to the virtual data center.<li>**DHCP.** Select this option to get an IP address dynamically where the network gateway acts as a DHCP router.</li><li>**Static IP Pool.** Select this option to get an IP address from a range of IP addresses configured for the network.</li><li>**Static Manual.** If you select this option, Cloud Application Manager finds the next available IP address from the virtual data center network and requests vCloud Air or vCloud Director to assign it to the VM.</li> |
-| Storage Profile | Choose a [storage option](http://vcloud.vmware.com/service-offering/block-storage) for vCloud Air.<li>**Standard.** Persistent block storage for occasional or event driven data access.</li><li>**SSD-Accelerated.** Higher performance block storage.</li> |
+```
+InstanceName-ServiceId-InstanceNumber
+```
 
-**Compute**
+In the Policy Box it's possible specify you want more that one instance. An example is “jdedepl-i8oq0-1”.  
 
-| Option | Description |
-|--------|-------------|
-| Virtual CPUs | Specify the number of CPUs. See the [VMware docs](http://pubs.vmware.com/vca/index.jsp?topic=%2Fcom.vmware.vca.od.ug.doc%2FGUID-6180A0A5-E390-449D-8FD6-CD8601554EAE.html) for vCloud Air VM CPU limit. |
-| Memory | Specify the virtual RAM in GB. See the [VMware docs](http://pubs.vmware.com/vca/index.jsp?topic=%2Fcom.vmware.vca.od.ug.doc%2FGUID-6180A0A5-E390-449D-8FD6-CD8601554EAE.html) for vCloud Air VM RAM limit. |
 
-**Disks**
+Once created this name, **modifying the vApp name is not supported**, because it could affect to the instance operations. However, it's possible to change manually this name on VMs and Hostnames.  
```

Comment:
```suggestion
Once deployed, **modifying the vApp name is not supported**, because it could affect the instance operations through Cloud Application Manager. However, it's possible to change this name manually on VMs and Hostnames.
```